### ISSUE TITLE
Accessiblity 

### ISSUE THREAD

**Issue by mumbereh** [mumbereh/To-do-list#5]
The code doesn't have any accessibility attributes like ARIA (Accessible Rich Internet Applications) label in both index.html & CSS, which could be added to improve accessibility for users with disabilities.

If this form is meant for user input, it's good practice to add labels for the input fields to provide context and improve accessibility.
